ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -61,4 +61,8 @@ message GetActiveContractsResponse {
   // The list of contracts that were introduced by the workflow with ``workflow_id`` at the offset.
   // Must be a valid LedgerString (as described in ``value.proto``).
   repeated CreatedEvent active_contracts = 3;
+
+  // A valid domain ID
+  // Optional Only present for multi-domain API. For 3.0 this will be mandatory.
+  string domain_id = 4;
```

Comment:
I think you want to incorporate the changes from #12863. A plain domain ID is not enough.

Reply:
What do you mean?

Reply:
We had settled on showing the transfer counter per domain on which the contract is active, not just a single domain ID.

Reply:
Ah, right. I got confused by the link that is pointing towards a random PR